src/components/Modal/MapOperatorToStakingProviderModal/index.tsx

```diff
@@ -109,58 +177,108 @@ const MapOperatorToStakingProviderModal: FC<
-              isOperatorMappedOnlyInRandomBeacon
-                ? mappedOperatorRandomBeacon
-                : isOperatorMappedOnlyInTbtc
-                ? mappedOperatorTbtc
-                : ""
-            }
-            onSubmitForm={onSubmit}
-            checkIfOperatorIsMappedToAnotherStakingProvider={
-              checkIfOperatorIsMappedToAnotherStakingProvider
-            }
-            mappedOperatorTbtc={mappedOperatorTbtc}
-            mappedOperatorRandomBeacon={mappedOperatorRandomBeacon}
-          />
-        </Box>
-        <AlertBox
-          status="warning"
-          withIcon={false}
-          alignItems="center"
-          p={"8px 10px"}
-        >
-          <AlertIcon h={"14px"} as={"div"} alignSelf="auto" />
-          <BodyXs>
-            Take note! tBTC + Random Beacon Apps Rewards Bundle will require two
-            transactions, one transaction per application.
-          </BodyXs>
-        </AlertBox>
+        {isEmptyOrZeroAddress(mappedOperatorTbtc) ||
+          (isEmptyOrZeroAddress(mappedOperatorRandomBeacon) && (
+            <>
+              <Box
+                p={"24px"}
+                border={"1px solid"}
+                borderColor={"gray.100"}
+                borderRadius={"12px"}
+                mt={"5"}
+                mb={"5"}
+              >
+                <LabelSm>
+                  {isEmptyOrZeroAddress(mappedOperatorTbtc)
+                    ? "tBTC"
+                    : isEmptyOrZeroAddress(mappedOperatorRandomBeacon)
+                    ? "Random Beacon"
+                    : "tBTC & Random Beacon"}
+                </LabelSm>
+                <StakeAddressInfo
+                  stakingProvider={account ? account : AddressZero}
+                />
+                <MapOperatorToStakingProviderForm
+                  innerRef={formRefTbtcRb}
+                  formId="map-operator-to-staking-provider-form-tbtc"
+                  initialAddress={
+                    isOperatorMappedOnlyInRandomBeacon
+                      ? mappedOperatorRandomBeacon
+                      : isOperatorMappedOnlyInTbtc
+                      ? mappedOperatorTbtc
+                      : ""
+                  }
+                  checkIfOperatorIsMappedToAnotherStakingProvider={async (
+                    operator: string
+                  ) => {
+                    const isOperatorAlreadyMappedTbtc =
+                      await checkIfOperatorIsMappedToAnotherStakingProvider(
+                        operator,
+                        "tbtc"
+                      )
+                    const isOperatorAlreadyMappedRandomBeacon =
+                      await checkIfOperatorIsMappedToAnotherStakingProvider(
+                        operator,
+                        "tbtc"
+                      )
+                    return (
+                      isOperatorAlreadyMappedTbtc &&
+                      isOperatorAlreadyMappedRandomBeacon
+                    )
```

Comment:
I think I've done a mistake here: https://github.com/threshold-network/token-dashboard/pull/625#discussion_r1429891983

We should use OR (`||`) operator here instead of AND {`&&`}. The condition should work the same as here:
https://github.com/threshold-network/token-dashboard/blob/17afd96148b72593c20d2c487bd79047f874a927/src/components/Modal/MapOperatorToStakingProviderModal/index.tsx#L79-L84
and we use the OR operator there.

```suggestion
                    return (
                      isOperatorAlreadyMappedTbtc ||
                      isOperatorAlreadyMappedRandomBeacon
                    )
```